pkg/cmd/subcommand/uninstall.go

```diff
@@ -169,49 +174,64 @@ func (i *Uninstall) RunUninstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 		return nil
 	}
 
-	grp, gctx := errgroup.WithContext(ctx)
-	defer gctx.Done()
+	grp, ctx := errgroup.WithContext(ctx)
+	defer done()
```

Comment:
You're sure to keep `defer done()` here?

Reply:
Yes, it needs to be removed.